components/dashboard.tsx

```diff
@@ -15,28 +15,36 @@ const Dashboard: FC<DashboardProps> = ({ siteUUID }) => {
   return (
     <div className="bg-ocf-black max-w-screen-lg w-screen min-h-screen px-4 mb-[75px]">
       <h1 className="mt-4 mb-6 text-ocf-gray text-3xl font-bold">Dashboard</h1>
-      <h2 className="mt-4 mb-2 text-ocf-gray text-base font-semibold">
-        Solar Activity
-      </h2>
-      <div className="flex flex-row w-full justify-center space-x-4">
-        <SunnyTimeframe siteUUID={siteUUID} />
-        <EnergyRecommendation siteUUID={siteUUID} />
-      </div>
-      <div className="flex flex-row w-full justify-start">
-        <ThresholdGraph siteUUID={siteUUID} />
-      </div>
-      <h2 className="mt-4 mb-2 text-ocf-gray text-base font-semibold">
-        Metrics
-      </h2>
-      <div className="flex flex-row w-full justify-start">
-        <ExpectedTotalOutput siteUUID={siteUUID} />
-      </div>
-      <div className="flex flex-row w-full justify-center space-x-4">
-        <CurrentOutput siteUUID={siteUUID} />
-        <CurrentCapacity siteUUID={siteUUID} />
-      </div>
-      <div className="flex flex-row w-full justify-start">
-        <Graph siteUUID={siteUUID} />
+      <div className="grid grid-areas-dashboard-mobile grid-cols-mobile-columns grid-rows-mobile-rows md:grid-areas-dashboard-desktop md:grid-cols-desktop-columns md:grid-rows-desktop-rows w-full gap-5">
+        <div className="grid-in-Heading1 block md:hidden">
+          <h2 className="mt-4 mb-2 text-ocf-gray text-base font-semibold">
+            Solar Activity
+          </h2>
+        </div>
+        <div className="grid-in-Sunny border-blue-200">
```

Comment:
I think you can remove the `border-blue-200` on this element and a few below it.